### DISCUSSION TITLE
Getting the true standard deviation on prediction

### DISCUSSION THREAD

**Discussion by DevergneTimothee** [deepmodeling/deepmd-kit#701] [Q&A]
Hello, 
I am using lammps, and it seems that when using several NN to perform simulations, the output in md.out for the energies is the max, min and avg deviation on the atomic energies predictions made by the different NN . Since the actual observable is the total energy, is it possible to have the standard deviation on the total energy during the simulation ? 
Best, Timothée

**Comment by njzjz** [comment]
<s>avg atomic energy std * number of atoms = total energy std</s>

**Reply by DevergneTimothee** [reply]
Hello, I think there is something I don't understand. 
If I take two NN trained on a system with two atoms, and I take the following atomic energy predictions: 
E_{11} = 1.1
E_{12} = -1.1
E_{21} = 0.9
E_{22} = -0.9 
Where the first subscript is for the NN and the second for the atoms. In both cases we have E_{tot} = 0 and, we have a total energy std of 0 (both the NN predict a total energy of 0).
But if you take the avg of atomic energy std, we have 
avg std energy = 0.5*(sqrt(0.5*((1.1-1.0)^2 +(0.9-1.0)^2)))  + sqrt(0.5*((-1.1+1)^2 +(-0.9+1)^2))) = 0.1 
Which is not the same as the standard deviation on the total energy. So there must be something I misunderstood.
Best
Timothée

**Reply by njzjz** [reply]
Oh you are right. So currently it's not supported.
